### DISCUSSION TITLE
YouTube有没有这样的插件啊

### DISCUSSION THREAD

**Discussion by LoneFireBlossom** [the1812/Bilibili-Evolved#1982] [提问 / Q&A] [answered]
如题，谷歌了一下没找到，又不可能在脚本库里漫无目的的搜索，所以想开个帖子问问。
我个人的需求其实就是倍速播放，以及屏蔽掉侧栏等无关内容。
虽然有那种专门倍速播放的插件不过不如这个插件这种形式好用。（而且YouTube自己的倍速播放竟然要点两次才能点到……真是麻烦）
屏蔽掉侧栏用广告屏蔽插件屏蔽元素就行，不过还是想如果有插件专门处理YouTube感觉更好一点。

**Comment by Coulomb-G** [comment]
不怎么用YouTube,也不会考虑给YouTube做类似的,你要是有需求可以自己写,这边是开源的

**Reply by LoneFireBlossom** [reply]
> 不怎么用YouTube,也不会考虑给YouTube做类似的,你要是有需求可以自己写,这边是开源的

啊这 我不是要求做= =只是问问大家有没有在用类似的现成插件
（我也希望我有朝一日能学会开发😭）

**Comment by the1812** [comment, marked as answer]
好像有些各站播放器通用的脚本来着, GreasyFork 上应该能找到

**Reply by LoneFireBlossom** [reply]
确实 在上面搜了一下找到了一个感觉不错，把B站链接排除了
https://greasyfork.org/zh-CN/scripts/381682-html5%E8%A7%86%E9%A2%91%E6%92%AD%E6%94%BE%E5%99%A8%E5%A2%9E%E5%BC%BA%E8%84%9A%E6%9C%AC

**Comment by RingingVolcano** [comment]
YouTube切倍速可以用快捷键的 Shift+,/.

**Comment by 1970633640** [comment]
曾经有个youtube center插件，很强大，功能可能比这个插件都全，可惜后来youtube改版后作者估计弃坑了
